src/app/styles/tailwind/ui/fontWeight.js

```diff
@@ -0,0 +1,6 @@
+const fontWeight = {
+  normal: '400',
+  bold: '700',
+};
+
+export default fontWeight;
```

Comment:
We dont need this, use font-bold, font-medium, .etc

Reply:
+